website/server/controllers/api-v3/chat.js

```diff
@@ -101,6 +111,22 @@ api.postChat = {
     let lastClientMsg = req.query.previousMsg;
     chatUpdated = lastClientMsg && group.chat && group.chat[0] && group.chat[0].id !== lastClientMsg ? true : false;
 
+    // Check for spam by seeing if the current user has posted too many messages in a short window
+    if (!(user.contributor.level >= SPAM_CONTRIBUTOR_LEVEL) && (group.type === 'party' || group.type === 'guild')) {
```

Comment:
I didn't know there were only two types so that is why :) now that I know that I can remove this check